scipy-1.0/covlett.tex

```diff
@@ -5,70 +5,100 @@
 
 % to adjust vertical spacing properties of
 % newlfm class see: https://tex.stackexchange.com/a/65533/43006
-\topmarginsize{2mm}
+%\unprtop{0mm}
+\topmarginsize{0mm}
 \topmarginskip{0in}
 \headermarginsize{0in}
 \headermarginskip{0in}
+\MinHead{0in}
+%\dateskipafter{0pt}
+%\dateskipbefore{0pt}
+\sigskipbefore{40pt}
+%\sigskipafter{0pt}
+\closeskipbefore{1pt}
+%\closeskipafter{0pt}
+%\addrfromskipbefore{0pt}
+\addrfromskipafter{0in}
+\addrtoskipbefore{0in}
+%\addrtoskipafter{0pt}
+\newlfmP{Headlinewd=0pt,Footlinewd=0pt}
+\newlfmP{sigsize=10pt}
+\bottommarginskip{10pt}
+\MinFoot{0pt}
+\leftmarginsize{0.95in}
+\rightmarginsize{0.95in}
 
 \usepackage{kpfonts}
+\usepackage{hyperref}
 \usepackage{url}
 
+\hypersetup{%
+  linkbordercolor=blue,% hyperlink borders will be blue
+  pdfborderstyle={/S/U/W 1}% border style will be underline of width 1pt
+}
 \widowpenalty=1000
 \clubpenalty=1000
 
-\newlfmP{headermarginskip=10pt}
-\newlfmP{sigsize=30pt}
-\newlfmP{dateskipafter=20pt}
-%\newlfmP{addrfromphone}
-%\newlfmP{addrfromemail}
-%\PhrPhone{Phone}
-%\PhrEmail{Email}
-
 \namefrom{Ralf Gommers,
           Matt Haberland,
           and Tyler Reddy}
 \addrfrom{%
     \today\\[10pt]
-    Quansight Labs,\\
+    Ralf Gommers\\
+    Quansight Labs\\
     The Netherlands\\
+    \texttt{ralf.gommers@gmail.com}\\
+    %phone number\\
     \\
-    BioResource and Agricultural Engineering,\\
-    California Polytechnic State University,\\
+    Matt Haberland\\
+    BioResource and Agricultural Engineering\\
+    California Polytechnic State University\\
     San Luis Obispo, CA, 93407, USA\\
+    \texttt{mhaberla@calpoly.edu}\\
+    %phone number\\
     \\
-    CCS-7 Applied Computer Science Group,\\
-    Los Alamos National Laboratory,\\
-    Los Alamos, NM, 87545, USA
+    Tyler Reddy\\
+    CCS-7 Applied Computer Science Group\\
+    Los Alamos National Laboratory\\
+    Los Alamos, NM, 87545, USA\\
```

Comment:
```suggestion
    Los Alamos, NM 87545, USA\\
```
Same for my address. Already correct for Rita.

Reply:
Otherwise LGTM

Reply:
ok, done for both addresses